File: samples/performance/SocketPro/perf_i.cs
public static class perfConst
{
	//defines for service CPerf
	public const uint sidCPerf = (SocketProAdapter.BaseServiceID.sidReserved + 1111);

	public const ushort idMyEchoCPerf = ((ushort)SocketProAdapter.tagBaseRequestID.idReservedTwo + 1);
	public const ushort idOpenRecordsCPerf = (idMyEchoCPerf + 1);
}

File: SproInstall/SocketPro/samples/performance/vbnet/PerformanceStudySocketPro/PerfSvr/Perf_i.vb
Public Class PerfConst
	'defines for service CPerf
    Public Const sidCPerf As Integer = (CInt(USOCKETLib.tagOtherDefine.odUserServiceIDMin) + 1111)

    Public Const idMyEchoCPerf As Short = (CShort(USOCKETLib.tagOtherDefine.odUserRequestIDMin) + 0)
	Public Const idOpenRecords As Short = idMyEchoCPerf + 1
End Class